test/integration/datatypes-in-results-test.ts

```diff
@@ -584,3 +584,90 @@ describe('Datatypes in results test', function() {
     ]
   });
 });
+
+describe('Collation LCID test', function() {
+  let connection: Connection;
+
+  beforeEach(function(done) {
+    connection = new Connection(config);
+
+    connection.on('errorMessage', function(error) {
+      console.log(`${error.number} : ${error.message}`);
+    });
+
+    connection.on('debug', function(message) {
+      if (debug) {
+        console.log(message);
+      }
+    });
+
+    connection.connect(done);
+  });
+
+  afterEach(function(done) {
+    if (!connection.closed) {
+      connection.on('end', done);
+      connection.close();
+    } else {
+      done();
+    }
+  });
+
+  it('should recieve correct column collation after sending different collation char and varchar', function(done) {
```

Comment:
typo: recieve => receive

```suggestion
  it('should receive correct column collation after sending different collation char and varchar', function(done) {
```